src/assets/custom-ion-icons/beanconqueror-reporting-parameter.svg

```diff
@@ -0,0 +1,8 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="45" height="45" viewBox="0 0 45 45">
```

Comment:
Where does this icone come from? Icons8? (Just don't want to have some licence violations